webviz_subsurface/containers/_summary_stats.py

```diff
@@ -7,41 +7,69 @@
 from webviz_config.webviz_store import webvizstore
 from webviz_config.common_cache import cache
 from ..datainput import scratch_ensemble
+from plotly.colors import DEFAULT_PLOTLY_COLORS
+import itertools
 
 
 class SummaryStats:
```

Comment:
All containers should now inherit the WebvizContainer class.
class SummaryStats(WebvizContainer)